README.md

```diff
@@ -1008,7 +1008,7 @@ Execution interfaces [`ISyncPolicy`](https://github.com/App-vNext/Polly/tree/mas
 
 ## Policy-kind interfaces: `ICircuitBreakerPolicy` etc
 
-Orthogonal to the execution interfaces, interfaces specific to the kind of Policy define properties and methods common to that type of policy.  
+Orthogonal to the execution interfaces, interfaces specific to the kind of policy define properties and methods common to that type of policy.  
```

Comment:
This sentence needs further thought in general, as I had to read it about 3 times to understand it.

Reply:
we should either use "Policy" or "policy" every where in the sentence but not both.